int32 feat_id
geometry_msgs/Point pt  # location of feature in frame
float32 depth # depth of the feature from the camera origin